pkg/bff/models/v1/errors.go

```diff
@@ -7,6 +7,8 @@ import (
 )
 
 var (
+	ErrInvalidField            = errors.New("invalid field")
```

Comment:
This feels a little too generic to me; I think it would be better to explain why a field is invalid. It seems like this is used for drop-down menu fields right now, though, so maybe it isn't a big deal.